pkgs/development/python-modules/lxml-stubs/default.nix

```diff
@@ -9,27 +9,34 @@
 
 buildPythonPackage rec {
   pname = "lxml-stubs";
-  version = "0.4.0";
+  version = "0.5.1";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "lxml";
     repo = "lxml-stubs";
-    rev = version;
```

Comment:
What's wrong with this?

Reply:
String literals should be avoided and going for the tag help during batch/mass updates if there is branch/tag confusion.